README.md

```diff
@@ -0,0 +1,57 @@
+# Hipo CI for iOS Initial Directory
+
+This repo contains Hipo's fastlane files for new iOS projects. In this documentation you can find how to use it and how to customize it.
+
+## Installation
+
+See the steps below for details about the installation.
+
+1. [Download Files](README.md#setup-dependencies)
+1. [Setup Dependencies](README.md#setup-dependencies)
+1. [Add project to Bitrise](README.md#add-project-to-bitrise)
+1. [Customization](README.md#customization)
+
+
+### Download Files
+You can directly download the files from **download branch** or directly [here](https://github.com/Hipo/ci-integration-ios/archive/download.zip) and copy items to project folder.
+
+### Setup Dependencies
+
+We prefer installing fastlane via *Bundler*. It also helps to track dependencies easily. So you can see *Gemfile* in the project.
+
+* Run `$ bundle update`
+
+It will install fastlane and other dependencies that we use to your project. **Make sure** that all files added to version control.  
+
+### Add project to Bitrise
+
+After you complete the installation, you need to setup Bitrise for that project. It will automatically understand the project contains fastlane integration.
```

Comment:
I think we should add its steps here in a way. It would be helpful to have a guide. It may be even a video.